MapboxDirections/MBAnnotation.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  MBAnnotationType.swift
+//  MapboxDirections
+//
+//  Created by Bobby Sudekum on 3/20/17.
+//  Copyright © 2017 Mapbox. All rights reserved.
```

Comment:
Omit this file header comment. This is a permissively licensed project under version control, so the comment adds little value.